packages/x-charts/src/models/axis.ts

```diff
@@ -451,7 +451,7 @@ export type PolarAxisDefaultized<
     triggerTooltip?: boolean;
   };
 
-export type AxisDefaultized<
+export type ComputedAxis<
```

Comment:
```suggestion
/**
 * The axis configuration with all computed attributes, like scale or sizing.
 */
export type ComputedAxis<
```

Reply:
Would it make sense to do not export it, and only export `ComputedXAxis` and `ComputedYAxis`?

It could make the distinction with polar axes easier in our internal codebase